Added short section about jcasc in the testing dev docu (#7650)
jenkins-infra/jenkins.io: content/doc/developer/testing/index.adoc

}
----

=== Verifying compatibility with Configuration as Code (JCasC)

The link:https://plugins.jenkins.io/configuration-as-code/[Configuration as Code plugin] is an opinionated way to configure Jenkins based on human-readable declarative configuration files.

> Writing such a file should be feasible without being a Jenkins expert, just translating into code a configuration process one is used to executing in the web UI.

Reviewer: ```suggestion
Writing such a file can be done without being a Jenkins expert by translating a configuration process one is familiar with executing in the web UI into code.
```
Adjust sentence structure and punctuation.

> If the plugin should support JCasC, it should also test the compatability.

Reviewer: ```suggestion
If the plugin supports JCasC, it also tests compatibility.
```
Suggestion to use present tense/avoid future voicing, update spelling. If this is incorrect, please disregard.

> More details can be found at the link:https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/PLUGINS.md[configuration-as-code-plugin documentation].

Reviewer: ```suggestion
More details can be found in the link:https://github.com/jenkinsci/configuration-as-code-plugin/blob/master/docs/PLUGINS.md[configuration-as-code-plugin documentation].
```
Suggestion to adjust wording for grammar.
